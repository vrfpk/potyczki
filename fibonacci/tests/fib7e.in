012187670855112
