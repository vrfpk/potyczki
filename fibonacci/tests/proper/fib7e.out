2249999999995734
